engine/api/repositories_manager.go

```diff
@@ -400,21 +353,16 @@ func getRepoFromRepositoriesManagerHandler(w http.ResponseWriter, r *http.Reques
 
 	client, err := repositoriesmanager.AuthorizedClient(db, projectKey, rmName)
 	if err != nil {
-		log.Warning("repositoriesManagerAuthorizeCallback> Cannot get client got %s %s : %s", projectKey, rmName, err)
-		return sdk.ErrNoReposManagerClientAuth
-
+		return sdk.WrapError(sdk.ErrNoReposManagerClientAuth, "repositoriesManagerAuthorizeCallback> Cannot get client got %s %s : %s", projectKey, rmName, err)
```

Comment:
getReposFromRepositoriesManagerHandler instead of repositoriesManagerAuthorizeCallback